Dockerfile

```diff
@@ -8,7 +8,7 @@ RUN go get github.com/tools/godep
 RUN go get code.google.com/p/go.tools/cmd/cover
 
 ENV GOPATH /go/src/github.com/docker/machine/Godeps/_workspace:/go
-ENV MACHINE_BINARY /go/src/github.com/docker/machine/docker-machine
+ENV MACHINE_BINARY /go/src/github.com/docker/machine/docker-machine/docker-machine
```

Comment:
There's a problem with `go get`. `go get` names the binary by whatever directory the `main` package is in. 

By putting it in a `docker-machine` dir, we can ensure that people will have a binary called `docker-machine`.